File: ios/Packages/SolennixCore/Sources/SolennixCore/Models/EventEquipment.swift
import Foundation

public struct EventEquipment: Codable, Identifiable, Sendable, Hashable {
    public let id: String
    public let eventId: String
    public let inventoryId: String
    public let quantity: Int
    public var notes: String?
    public let createdAt: String
    // Denormalized joined fields
    public var equipmentName: String?
    public var unit: String?
    public var currentStock: Double?
}

File: android/core/model/src/main/java/com/creapolis/solennix/core/model/EventEquipment.kt
package com.creapolis.solennix.core.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class EventEquipment(
    val id: String = "",
    @SerialName("event_id") val eventId: String = "",
    @SerialName("inventory_id") val inventoryId: String,
    val quantity: Int,
    val notes: String? = null,
    @SerialName("created_at") val createdAt: String = "",
    @SerialName("equipment_name") val equipmentName: String? = null,
    val unit: String? = null,
    @SerialName("current_stock") val currentStock: Double? = null
)
